src/Components/MovieList/index.tsx

```diff
@@ -27,17 +34,37 @@ const initialState: MovieListLocalState = {
 }
 
 const MovieList: React.FC = () => {
-    const [fetchedData, setFetchedData] = useState<MoviesResponseModel>()
+    const [localState, setLocalState] = useReducer(reducer, initialState)
+    const dispatch = useDispatch()
 
-    useEffect(() => {
-        const data = useFetch<MoviesResponseModel>('url to movies api')
-        setFetchedData(data)
-    }, [])
+    const movies = useSelector(Selectors.getMoviesSelector)
+    const sortBy = useSelector(Selectors.getSotrtBySelector)
+    const sortOrder = useSelector(Selectors.getSortOrderSelector)
+    const genre = useSelector(Selectors.getGenreSelector)
+    const search = useSelector(Selectors.getSearchSelector)
+    const searchBy = useSelector(Selectors.getSearchBySelector)
 
-    const [state, dispatch] = useReducer(reducer, initialState)
+    useEffect(() => {
+        const params: Record<string, string> = {
+            sortBy,
+            sortOrder,
+            search,
+            searchBy,
+            filter: genre,
+            limit: '12',
```

Comment:
it will be better to put this into constant

Reply:
Done